packages/core/src/Stores/client-store.js

```diff
@@ -613,6 +613,13 @@ export default class ClientStore extends BaseStore {
     async init(login_new_user) {
         this.setIsLoggingIn(true);
         const authorize_response = await this.setUserLogin(login_new_user);
+
+        // On case of invalid token, no need to continue with additional api calls.
+        if (authorize_response?.error) {
+            this.setIsLoggingIn(false);
```

Comment:
I think we will also need to emit with `setInitialized` and `setSwitched` too, right?

Reply:
Added. thanks.